python-sdk/src/astro/files/base.py

```diff
@@ -112,6 +115,8 @@ def export_to_dataframe(self, **kwargs) -> pd.DataFrame:
         """Read file from all supported location and convert them into dataframes."""
         mode = "rb" if self.is_binary() else "r"
         with smart_open.open(self.path, mode=mode, transport_params=self.location.transport_params) as stream:
+            # Update kwargs with the pandas kwargs supplied by user in File object
```

Comment:
In the case of Snowflake and Postgres implementations, we would need to accept this parameter and pass it to this function from where this function is getting called. 
https://github.com/astronomer/astro-sdk/blob/0e9d07b53bb2218dabecdbb991dd49cc2ff94cfe/python-sdk/src/astro/databases/snowflake.py#L537

https://github.com/astronomer/astro-sdk/blob/0e9d07b53bb2218dabecdbb991dd49cc2ff94cfe/python-sdk/src/astro/databases/postgres.py#L208